playbooks/roles/ocp-config/tasks/main.yaml

```diff
@@ -61,10 +58,24 @@
     line: '\1mastersSchedulable: False'
     backrefs: yes
 
-- name: Create Chrony machineconfig files for coreOS
+- name: Setup time synchronisation with Chrony
   import_tasks: chrony.yaml
   when: chronyconfig.enabled
 
+- name: Stat ../../ocp4-helpernode/machineconfig/
+  stat:
```

Comment:
Make sense to move these two tasks to chrony.yaml?

Reply:
It is in the main.yaml, not only specific to chrony.
If helpernode add other machineconfig files in the future, they should be located into this directory.
Today, it is only chrony, but we never know ... what's your opinion about it ?

Reply:
See other comment. We want to be specific to chrony here IMO.

Reply:
done